recipes/kalasiris/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "kalasiris" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5bdcb44b71dd7c3a0cf3deb42d18a7516098cb2a372710433413c0d1ac9449db
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  commands:
+    - test -f $PREFIX/kalasiris/kalasiris.py
```

Comment:
I expect this doesn't get installed at this path.

As you have a `run_tests.py` file you can delete the entire `test` section as it should be ran automatically.

Reply:
That's a good point.  I pushed again before eliminating this.  But will do so.